back-end/helpers/db-helpers.js

```diff
@@ -24,7 +20,38 @@ async function getRowFromDb(queryString) {
   return results.rows[0];
 };
 
+async function getUserByEmail(email) {
```

Comment:
We can probably move any actual queries to another file - potentially if we have to add more tables and more queries - this file could become huge

Reply:
👍 I went back and forth on whether to do that, but figured I might wait until we had more functions here. Can do it now!